shared/bash_remediation_functions/perform_audit_adjtimex_settimeofday_stime_remediation.sh

```diff
@@ -2,7 +2,7 @@ source fix_audit_syscall_rule.sh
 
 # Perform the remediation for the 'adjtimex', 'settimeofday', and 'stime' audit
 # system calls on Red Hat Enterprise Linux 7 or Fedora OSes
```

Comment:
Updating this comment might be nice.

Reply:
@cipherboy, sounds reasonable. Please find updated description.